apps/wallet/src/ui/app/pages/site-connect/index.tsx

```diff
@@ -29,17 +34,20 @@ function SiteConnectPage() {
 	);
 	const dispatch = useAppDispatch();
 	const permissionRequest = useAppSelector(permissionSelector);
-	const activeAddress = useActiveAddress();
-	const [accountsToConnect, setAccountsToConnect] = useState<string[]>(() =>
-		activeAddress ? [activeAddress] : [],
+	const activeAccount = useActiveAccount();
+	const { data: accounts } = useAccounts();
+	const unlockedAccounts = accounts?.filter((account) => !account.isLocked) ?? [];
```

Comment:
`isLocked` means the account is locked for write. Accounts also are able to be read (un)locked. In this case if we want to be strict checking `isLocked` is fine (I am ok with this - just checking)